site/pages/view-all-disruptions.page.tsx

```diff
@@ -362,6 +379,30 @@ const ViewAllDisruptions = ({ disruptions, services, newDisruptionId }: ViewAllD
         ); // eslint-disable-next-line react-hooks/exhaustive-deps
     }, [selectedServices]);
 
+    useEffect(() => {
+        const operatorsToSet = selectedOperatorsNocs.map(
+            (selOpNoc) => operators.find((op) => op.nocCode === selOpNoc) as Operator,
+        );
+        const disruptionOperatorsToSet: DisruptionOperator[] = operatorsToSet.map((op) => {
+            return {
+                operatorName: op.operatorPublicName,
+                operatorRef: op.nocCode,
+            };
+        });
+        setFilter({
+            ...filter,
+            operators: disruptionOperatorsToSet,
+        });
+        // eslint-disable-next-line react-hooks/rules-of-hooks
+        useFiltersOnDisruptions(
```

Comment:
Renaming `useFiltersOnDisruptions` to a name without prefix `use` will allow you to remove the eslint disable comment

Reply:
Ah good catch! Thanks